File: sdkwork-app-sdk-csharp/Models/FaqDetailVO.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace App.Models
{
    public class FaqDetailVO
    {
        public string? CreatedAt { get; set; }
        public string? UpdatedAt { get; set; }
        public string? Id { get; set; }
        public string? Question { get; set; }
        public string? Answer { get; set; }
        public string? CategoryId { get; set; }
        public string? CategoryName { get; set; }
        public int? ViewCount { get; set; }
        public int? HelpfulCount { get; set; }
    }
}

File: sdkwork-app-sdk-kotlin/src/main/kotlin/com/sdkwork/app/FaqDetailVO.kt
package com.sdkwork.app

data class FaqDetailVO(
    val createdAt: String? = null,
    val updatedAt: String? = null,
    val id: String? = null,
    val question: String? = null,
    val answer: String? = null,
    val categoryId: String? = null,
    val categoryName: String? = null,
    val viewCount: Int? = null,
    val helpfulCount: Int? = null
)
